lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
```

Comment:
Any reason we're using `lerna` rather than npm workspaces?

Reply:
We're only using lerna here for package publishing because npm workspaces doesn't yet support the clean publishing flow that lerna offers.